catalog/message/schema/catalog-schema.json

```diff
@@ -1,39 +1,338 @@
+                        "$ref": "#/definitions/Multilanguage"
+                    }
+                },
+                "dct:identifier": {
+                    "type": "string"
+                },
+                "dct:isReferencedBy": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:issued": {
+                    "$ref": "#/definitions/time"
+                },
+                "dct:language": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:license": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:modified": {
+                    "$ref": "#/definitions/time"
+                },
+                "dct:publisher": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:relation": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:title": {
+                    "type": "string"
+                },
+                "dct:type": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "odrl:hasPolicy": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "https://raw.githubusercontent.com/International-Data-Spaces-Association/ids-specification/dcat-catalog-schema-and-shape/negotiation/message/schema/contract-schema.json#/definitions/Policy"
+                    },
+                    "minItems": 1
+                }
+            }
+        },
+        "DataService": {
+            "type": "object",
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                },
+                {
+                    "properties": {
+                        "dcat:endpointDescription": {
+                            "$ref": "#/definitions/Reference"
+                        },
+                        "dcat:endpointURL": {
+                            "$ref": "#/definitions/Reference"
+                        },
+                        "dcat:servesDataset": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "https://w3id.org/dspace/schemas/v0.8/dataset-schema.json#/definitions/Dataset"
+                            },
+                            "minItems": 1
+                        }
+                    }
+                }
+            ]
+        },
+        "Distribution": {
+            "type": "object",
+            "properties": {
+                "dcat:accessService": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/DataService"
+                    },
+                    "minItems": 1
+                },
+                "dcat:accessURL": {
+                    "$ref": "#/definitions/Reference"
+                },
```

Comment:
This isn't in the context yet, but might actually fit well in the context with `"@type": "@id"` given that it is always just an URL

Reply:
Proposal: After we have finished this PR, let's block us one hour and together put more properties into the context file.